history-of-quakers/modernized/21-chapter-19.adoc

```diff
@@ -101,14 +117,21 @@ Oh! for the Lord`'s sake, do something that may roll away this reproach;
 which that you may, is the prayers of your brother,
 who could be contented to write himself,
 
+[.signed-section-closing]
 Your companion in tribulation,
 
-JEREMY IVESJan.
-14, 1660.
+[.signed-section-signature]
+Jeremy Ives
+
+[.signed-section-context-close]
+Jan. 14, 1660.
 
+[.postscript]
```

Comment:
Not sure if this should be [.postscript].

Reply:
seems right to me. 👍